### PR TITLE
aura airdrop claim script

### PR DESCRIPTION
**Pull request by jalbrekt85**
handles #561 

```sh
 brownie run issue/561/claim_airdrops main <msig address>
```

### CODE REVIEW

File: scripts/issue/561/claim_airdrops.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+from helpers.addresses import r
+from brownie import interface
+import json
+
+
+"""
+eligble:
+treasury voter: 0xA9ed98B5Fb8428d68664f3C5027c62A10d45826b
+ibbtc msig: 0xB76782B51BFf9C27bA69C77027e20Abd92Bcf3a8
+ibbtc peak (todo: redirect): 0x41671BA1abcbA387b9b2B752c205e22e916BE6e3
```
Comment (jorijnsmit) [inline scripts/issue/561/claim_airdrops.py:11]: there is no proof json for this address, does this mean the redirect was successful and its claimable amount is indeed 0?

**Review by jorijnsmit — COMMENTED** [review]
indeed what @petrovska-petro said, autolock the voter's claim and transfer full amount to voter after claim for the other msigs.

File: scripts/issue/561/claim_airdrops.py (hunk 1 above)
Comment (petrovska-petro) [reply scripts/issue/561/claim_airdrops.py:11]: i think for the peak was redirected to the treasury_voter. [ref](https://github.com/aurafinance/aura-token-allocation/blob/6e95bc02fe6a1d401a7d4fbdf8634f49e21f6738/redirection/badger.json#L3)

Comment (jorijnsmit) [reply scripts/issue/561/claim_airdrops.py:11]: damn if so easy why didnt we redir all to voter..?

File: scripts/issue/561/claim_airdrops.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    with open('scripts/issue/561/airdrop_list.json', 'r') as f:
+        airdrop_list = json.load(f)
+
+    if not aura_airdrop.hasClaimed(msig_address):
+        msig.take_snapshot(tokens=[aura])
+
+        with open(f'scripts/issue/561/proofs/{msig_address}.json', 'r') as f:
+            proof = json.load(f)['account']
+
+        is_voter = True if msig_address == r.badger_wallets.treasury_voter_multisig else False
```
Comment (jorijnsmit) [inline scripts/issue/561/claim_airdrops.py:33]:
```suggestion
        is_voter = msig_address == r.badger_wallets.treasury_voter_multisig
```

**Review by jorijnsmit — COMMENTED** [review]
lgtm as is, but you have a merge conflict so will approve after that is solved

**Review by jorijnsmit — APPROVED** [review]

**Review by petrovska-petro — APPROVED** [review]
LGTM!

**Pull request merged**